### PR TITLE
⚡ word count feature added

### PR DESCRIPTION
**Pull request by ramsharma-prog**
# Description
Added "Max words" count feature to Multiline text component, count words instead of characters in the Multiline text field

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Primarily involves passing new attributes into the already existing nunjucks govukInput macro.

Currently, we test that the designer suffix and prefix fields have the correct help text.

 I would like to test that when the options are passed to the runner they render correctly, kindly advise me on where I can find a similar example of this being tested.
[](url)
# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and versioning
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto main and there are no code conflicts
- [x] I have checked deployments are working in all environments
- [x] I have updated the architecture diagrams as per Contribute.md

image

### CODE REVIEW

**Comment by RichardBray** [comment]
@ramsharma-prog would it be possible to link to the issue in this PR's description?

**Review by jenbutongit — APPROVED** [review]
LGTM!

File: smoke-tests/designer/wdio.conf.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const { hooks } = require("./support/hooks");
 const drivers = {
-  chrome: { version: "100.0.4896.60" }, // https://chromedriver.chromium.org/
+  chrome: { version: "101.0.4951.54" }, // https://chromedriver.chromium.org/
```
Comment (ramsharma-prog) [inline smoke-tests/designer/wdio.conf.js:3]: Chrome driver updated

**Comment by jenbutongit** [comment]
sorry @ramsharma-prog - you might need to update chrome driver again

**Pull request merged**